docs/content/architecture/scheduling.md

```diff
@@ -41,68 +41,52 @@ environments, they can be customized or removed as needed. Assuming a PostgresCl
 named 'hippo', the default Pod Topology Spread Constraints applied on Postgres
 Instance and pgBackRest Repo Host Pods are as follows:
 
-```
-    topologySpreadConstraints:
```

Comment:
📝 This looks like it moved because I changed its indentation to match a similar block below.